### PR TITLE
Feat: Add Resizable bar, input and card components

### PR DESCRIPTION
**Pull request by marihemori**
Added a base layout. Added Resizable bar component and other components for it, input and cards.

image

image

### CODE REVIEW

File: src/layouts/InsightsLayout/index.vue [hunk 1]
```diff
@@ -27,7 +31,7 @@ export default {
   overflow: hidden;
 
   & > * {
-    padding: $unnnic-spacing-sm;
+    padding: 0 16px 0 16px;
```
Comment (Aldemylla) [inline src/layouts/InsightsLayout/index.vue:34]:
```suggestion
    padding: 0 $unnnic-spacing-sm;
```
- Repeating the px becomes unnecessary in this case, as the padding property supports shortcuts to use only two values (horizontal and vertical);
- Always try to use tokens from our design system to maintain security and ease if changes are necessary.

File: src/views/insights/Home/index.vue [hunk 2]
```diff
@@ -2,15 +2,13 @@
...
-        Hoje
-        <img src="/src/assets/images/icons/expand_more.png" alt="" />
-      </button>
+      <unnnic-button
+        text="Dashboards"
+        icon-left="bar_chart_4_bars"
+        type="tertiary"
+        size="small"
+      />
+      <!-- <unnnic-input-date-picker size="sm"></unnnic-input-date-picker> -->
```
Comment (Aldemylla) [inline src/views/insights/Home/index.vue:11]:
```suggestion
      
```
Comments with unused code make the code dirty

File: src/components/ResizableBar.vue [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+  background-color: $unnnic-color-neutral-white;
+  width: 100%;
+  height: 210px;
+
+  &__header {
+    display: flex;
+    flex-direction: row;
+    place-content: space-between;
+    align-items: center;
+    margin-bottom: 2rem;
```
Comment (Aldemylla) [inline src/components/ResizableBar.vue:172]:
```suggestion
    margin-bottom: $unnnic-spacing-lg;
```

File: src/components/ResizableBar.vue [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+        display: grid;
+        grid-template-columns: repeat(2, 2fr);
+        grid-template-rows: repeat(2, 2fr);
+        grid-column-gap: $unnnic-spacing-sm;
+        grid-row-gap: $unnnic-spacing-sm;
+
+        .more-prompts-card {
+          background-color: $unnnic-color-neutral-soft;
+          color: $unnnic-color-neutral-darkest;
+          font-size: 20px;
```
Comment (Aldemylla) [inline src/components/ResizableBar.vue:246]:
```suggestion
          font-size: $unnnic-font-size-title-sm;
```

File: src/components/ResizableBar.vue [hunk 5]
```diff
@@ -0,0 +1,376 @@
...
+          <div class="insights-list">
+            <ul>
+              <li>Recentes</li>
+              <li>Segmentos</li>
+              <li>Varejo</li>
+              <li>Weni plataforma</li>
+              <li class="selected">Atendimento humano</li>
+            </ul>
+          </div>
+          <div class="insights-cards">
```
Comment (Aldemylla) [inline src/components/ResizableBar.vue:32]: There is a lot of repeated code in this section, would it be viable to transform the cards into reusable components?

**Review by Aldemylla — CHANGES_REQUESTED** [review]

File: src/components/ResizableBar.vue (hunk 5 above)
Comment (marihemori) [reply src/components/ResizableBar.vue:32]: Of course, it would be great to avoid repeated code.

File: src/components/InsightsInput.vue [hunk 6]
```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="insights-input">
+    <input
+      class="insights-input__input"
+      type="text"
+      placeholder="Peça insights ao InsightsGPT..."
+    />
+    <button class="insights-input__button">
+      <img src="../assets/images/icons/send.png" alt="" />
+    </button>
```
Comment (Aldemylla) [inline src/components/InsightsInput.vue:10]:
```suggestion
    <unnnic-button
      class="insights-input__button"
      type="secondary"
      size="large"
      iconCenter="send"
    />
```

File: src/components/ResizableBar.vue [hunk 7]
```diff
@@ -0,0 +1,319 @@
...
+      height: `${Math.min(sidebarHeight, maxHeight)}px`,
+    }"
+  >
+    <header class="resizable-bar__header" @mousedown="startResizing">
+      <hr
+        :class="{ 'green-color': isResizing }"
+        class="resizable-bar__separator"
+      />
+      <button @click="resizeBar" class="resizable-bar__circle-up">
+        <img src="../assets/images/icons/expand_circle_up.png" alt="Seta" />
```
Comment (Aldemylla) [inline src/components/ResizableBar.vue:15]:
```suggestion
       <unnnic-icon
         icon="expand_circle_up"
         size="md"
         scheme="neutral-cleanest"
         clickable
       />
```

**Review by Aldemylla — CHANGES_REQUESTED** [review]

**Review by Aldemylla — APPROVED** [review]

**Pull request merged**